src/three/CMPTLoader.js

```diff
@@ -60,6 +63,18 @@ export class CMPTLoader extends CMPTLoaderBase {
 
 				}
 
+				// 3DTILES_content_gltf
```

Comment:
Good question! I made the assumption that it should be taken into account anywhere that a tile could be loaded, but that may have been based on incomplete reading, and cesium as a ref, where they use the magic as a key into their loader factory everywhere as far as I saw, so adding support to something in their loader factory added support to it in other places also?

Though that is likely too naive of a reading, the implicit tiling looked much more complicated?

Reply:
I think I prefer following a reasonable interpretation of the spec here. The fact that Cesium affords loading GLTF from within CMPT seems like a side effect of their implementation rather than by design. Cesium also allows you to load GLTF 1.0 files within the B3DM files which isn't valid by the definition of the format, for example, but it works due to their model loading architecture. In fact there was one issue already where a user thought GLTF 1.0 was valid causing an error so I want to follow the spec instead of matching Cesium's behavior. And given that the listing of the extension is in the tileset.json and not in the CMPT file it seems that the tile set is the only place it would apply.

Reply:
Looks like there is a [migration guide](https://github.com/CesiumGS/3d-tiles/blob/main/extensions/3DTILES_content_gltf/MIGRATION_GUIDE.md#composite-cmpt) that I missed before? It states inner contents of a .cmpt can be gltf

Reply:
> Looks like there is a migration guide that I missed before? It states inner contents of a .cmpt can be gltf.

I don't see where it says the inner contents can be GLTF? Perhaps I'm interpreting the file incorrectly but that README looks to me like it's providing instructions on how to convert the older file formats (B3DM, I3DM, PNTS, CMPT) to GLTF without any feature loss.

> It also includes some specific info regarding when to apply RTC_CENTER which I haven't looked into further yet (!?)

I believe I implemented B3DM RTC_CENTER with some sample model another user was having a problem with though there could still be an issue. It hasn't been implemented for RTC_CENTER yet, though. I haven't seen any sample data available for it.